src/library/rekordbox/rekordboxfeature.cpp

```diff
@@ -825,12 +978,26 @@ TreeItemModel* RekordboxFeature::getChildModel() {
 
 QString RekordboxFeature::formatRootViewHtml() const {
     QString title = tr("Rekordbox");
-    QString summary = tr("Reads playlists and folders from Rekordbox prepared removable devices.");
+    QString summary = tr("Reads the following from Rekordbox prepared removable devices:");
+    QString item1 = tr("Playlists");
```

Comment:
Use QStringList instead of a bunch of named strings. Then you can loop over the QStringList below for the HTML.